content/applications/essentials/activities.rst

```diff
@@ -2,237 +2,348 @@
+<activities/chatter>` thread, :ref:`Kanban <activities/kanban>` view, :ref:`list <activities/list>`
+view, or :ref:`activities <activities/activity>` view of an application.
 
-.. tip::
-   Individual applications have a list of *Activity Types* dedicated to that application. For
-   example, to view and edit the activities available for the *CRM* application, go to
-   :menuselection:`CRM app --> Configuration --> Activity Types`.
+.. _activities/chatter:
 
-Enter a title for the activity in the :guilabel:`Summary` field, located in the :guilabel:`Schedule
-Activity` pop-up window.
+Chatter
+-------
 
-To assign the activity to a different user, select a name from the :guilabel:`Assigned to` drop-down
-menu. Otherwise, the user creating the activity is automatically assigned.
+Activities can be created from the *chatter* on any record.
 
-Lastly, feel free to add any additional information in the optional :guilabel:`Log a note...` field.
+To schedule a new activity, click the :guilabel:`Activities` button, located at the top of the
+chatter. In the :guilabel:`Schedule Activity` pop-up window that appears, :ref:`fill out the
+Schedule Activity form <activities/form>`.
 
-.. note::
-   The :guilabel:`Due Date` field on the :guilabel:`Schedule Activity` pop-up window auto-populates
-   based on the configuration settings for the selected :guilabel:`Activity Type`. However, this
-   date can be changed by selecting a day on the calendar in the :guilabel:`Due Date` field.
+.. image:: activities/chatter.png
+   :align: center
+   :alt: New activity type form.
 
-Lastly, click one of the following buttons:
+.. _activities/kanban:
 
-- :guilabel:`Schedule`: adds the activity to the chatter under :guilabel:`Planned activities`.
-- :guilabel:`Mark as Done`: adds the details of the activity to the chatter under :guilabel:`Today`.
-  The activity is not scheduled, it is automatically marked as completed.
-- :guilabel:`Done \& Schedule Next`: adds the task under :guilabel:`Today` marked as done, and
-  opens a new activity window.
-- :guilabel:`Discard`: discards any changes made on the pop-up window.
+Kanban view
+-----------
 
-.. image:: activities/schedule-pop-up.png
-   :alt: View of CRM leads and the option to schedule an activity.
+Another way that activities are created is from the *Kanban* view in an application.
 
-.. note::
-   Depending on the activity type, the :guilabel:`Schedule` button may be replaced by a
-   :guilabel:`Save` button, or an :guilabel:`Open Calendar` button.
+When in an application, click the :guilabel:`(Kanban)` icon, represented by two bars beneath a
+horizontal lime, in the top-right corner, and the data is displayed in a Kanban format.
 
-Scheduled activities are added to the chatter for the record under :guilabel:`Planned activities`.
+.. image:: activities/kanban.png
+   :align: center
+   :alt: The Kanban icon in the top menu, highlighted.
```

Comment:
I disagree only because this is the essentials doc, and the icons are not always called out/are not always intuitive. I know you cna read them when you mouse over it, but I know I wouldn't pick out this icon on my own. I do like keeping this in, since I think it is of value.